src/main/java/uk/gov/hmcts/reform/demo/controllers/RootController.java

```diff
@@ -25,4 +26,14 @@
     public ResponseEntity<String> welcome() {
         return ok("Welcome to spring-boot-template");
     }
+
+    /**
+     * Dummy endpoint, that demonstrates how the Global Exception handler can be used to capture
+     * and parse exceptions into a standard format.
+     * @return A ResponseEntity
+     */
+    @GetMapping("/StoreFile")
```

Comment:
Due to it only being a Dummy endpoint, feel free to ignore this one :) However, REST conventions would indicate the endpoint name should be lower case, and potentially just '/file', if the endpoint is just around getting files

Reply:
Yes, you are right. I have changed the end point.